amethyst_renderer/src/pipe/effect/mod.rs

```diff
@@ -115,7 +115,7 @@ impl Effect {
     }
 
     /// FIXME: Update raw buffer without transmute.
-    pub fn update_constant_buffer<N, T>(&self, name: N, data: &T, enc: &mut Encoder)
+    pub fn update_constant_buffer<N, T>(&mut self, name: N, data: &T, enc: &mut Encoder)
```

Comment:
Same as above.

Reply:
I know it isn't required by compiler but you don't want to update the same constant buffer simultaneously.

Reply:
I know it isn't required by compiler but you don't want to update the same constant buffer simultaneously.

Reply:
👍 Gotcha thanks